### PR TITLE
Remove explicit :port in request-base-uri

### PR DESCRIPTION
**Pull request by rymndhng**
This function is used for extracting the base URI of the original client request and generally for building URIs relative to the client's base URL. We should not explicitly set the :port unless the client specifies it as part of the `x-fowarded-host` or host headers.

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -3,18 +3,18 @@
   :url "https://www.github.com/unbounce/treajure"
   :license {:name "The MIT License (MIT)"
             :url "http://opensource.org/licenses/MIT"
-            :comments "Copyright (c) 2015 Unbounce Marketing Solutions Inc."}
```
Comment (ddossot) [inline project.clj]: Better use a range `2015-2016` to keep the inception year.

**Comment by ddossot** [comment]
This change breaks backwards compatibility. I suggest that you bump the version to `2.0.0-SNAPSHOT` to indicate a major change has occurred.

File: project.clj (hunk 1 above)
Comment (rymndhng) [inline project.clj]: good point :)

**Comment by rymndhng** [comment]
From further investigation, I am not going to pursue the changes in this diff.

With multiple ELBs configured: `client -> ELB -> router -> ELB -> backend`, the backend cannot reliably use `x-forwarded-port` and `x-forwarded-scheme` headers since the ELB will overwrite the messages when configured .

I will repurpose this diff to bump some dependencies though :)

**Pull request closed without merging**